ChangeLog

```diff
@@ -2,15 +2,21 @@
 Pylint's ChangeLog
 ------------------
 
-
-
-What's New in Pylint 2.8.3?
+What's New in Pylint 2.9.0?
 ===========================
-Release date: 2021-04-26
+Release date: TBA
 
 ..
   Put new features and bugfixes here and also in 'doc/whatsnew/2.9.rst'
 
+* The warning for ``arguments-differ`` now signals explicitly the difference it detected
+  by naming the argument or arguments that changed and the type of change that occurred.
+
+
+What's New in Pylint 2.8.3?
+===========================
+Release date: TBA
```

Comment:
@Pierre-Sassoulas Should I remove the `2.8.3` section?

Reply:
I think we're going to release a 2.8.3 next. It depends on how long the blocker takes ?